index.html

```diff
@@ -75,136 +78,219 @@ <h3 class="title">My Recent Works</h3>
-                <div class="category-list">Ruby</div>
-                <div class="category-list">CSS</div>
-                <div class="category-list">HTML</div>
-              </div>
+            <div class="category-list-container">
+              <div class="category-list">Javascript</div>
+              <div class="category-list">Ruby</div>
+              <div class="category-list">CSS</div>
+              <div class="category-list">HTML</div>
+            </div>
           </li>
           <li class="frameworks">
             Frameworks
-              <div class="category-list-container">
-                <div class="category-list">React.js</div>
-                <div class="category-list">Ruby on Rails</div>
-                <div class="category-list">RSpec</div>
-                <div class="category-list">Capybara</div>
-              </div>
+            <div class="category-list-container">
+              <div class="category-list">React.js</div>
+              <div class="category-list">Ruby on Rails</div>
+              <div class="category-list">RSpec</div>
+              <div class="category-list">Capybara</div>
+            </div>
           </li>
           <li class="skills">
             Skills
-              <div class="category-list-container">
-                <div class="category-list">Database Management</div>
-                <div class="category-list">Version Control</div>
-                <div class="category-list">CLI</div>
-                <div class="category-list">API Design</div>
-                <div class="category-list">Web Development</div>
-              </div>
+            <div class="category-list-container">
+              <div class="category-list">Database Management</div>
+              <div class="category-list">Version Control</div>
+              <div class="category-list">CLI</div>
+              <div class="category-list">API Design</div>
+              <div class="category-list">Web Development</div>
+            </div>
           </li>
         </ul>
       </div>
     </section>
+
+
+    <!--  contact form section -->
+    <!-- added by habtamualemayehu16@gmail.com   -->
+    <section class="reg-form">
+      <div class="form-body">
+        <div class="header-text">
+          <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+        </div>
+        <form action="https://formspree.io/f/xknaqwwd" method="post">
+          <input type="text" class="text-field" placeholder="Full name" min="1" max="30" name="Name" required>
+          <input type="email" class="text-field" placeholder="Email address" name="Email address" required>
+          <textarea name="Bio" id="bio" class="text-field" cols="30" rows="5" min="1" max="500" placeholder="Write me something"></textarea required>
+          <div class="form-btn">
+              <button type="submit" class="about-me-btn">Get in touch</button>
```

Comment:
- [ ] Your button currently doesn't have the right button interaction styles as shown in the Figma design. So kindly fix this as well :100: 
- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/228594977-62417279-f315-4b43-b4fe-97c93e08b817.png)